test/libsolidity/semanticTests/tryCatch/invalid_error_encoding.sol

```diff
@@ -153,15 +153,15 @@ contract C {
 // ----
 // f1() -> 2
 // f1a() -> 2
-// f1b() -> FAILURE, hex"12345678", 0x0, 0, "\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0"
+// f1b() -> FAILURE, hex"12345678", hex"0000000000000000000000000000000000000000000000000000000000000000", hex"0000000000000000000000000000000000000000000000000000000000000000", hex"0000000000000000000000000000000000000000000000000000000000000000"
 // f1c() -> 2
 // f2() -> 2
 // f2a() -> 2
-// f2b() -> FAILURE, hex"08c379a0", 0x100, 0, "\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0"
+// f2b() -> FAILURE, hex"08c379a0", 0x0100, 0, ""
 // f2c() -> 1
 // f3() -> 2
 // f3a() -> 2
-// f3b() -> FAILURE, hex"08c379a0", 0x20, 48, "\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0"
+// f3b() -> FAILURE, hex"08c379a0", 0x20, 0x30, ""
```

Comment:
This is interesting, that we consider `hex"0000000000000000000000000000000000000000000000000000000000000000"` and `""` the same.

Reply:
What about this one? Fine to add to #11277?